acceptance/acceptance_test.go

```diff
@@ -142,6 +142,20 @@ func TestAcceptance(t *testing.T) {
 		}
 	}
 
+	suiteManager = &SuiteManager{out: t.Logf}
+	suite := spec.New("acceptance suite", spec.Report(report.Terminal{}))
+
+	suite("p_current", func(t *testing.T, when spec.G, it spec.S) {
```

Comment:
Correct me if I'm wrong but it would appear that these would always execute regardless of ACCEPTANCE_SUITE_CONFIG. For example, in cases where we want to test (pack=previous, cb_pack=\*, lifecycle=\*) it would run the standard suite of tests against the current version of pack even though it's not what we requested.

Reply:
Ah, this is true... I'll add a check here. For the value of `ACCEPTANCE_SUITE_CONFIG` that you describe, I believe the desired behavior is that we skip these tests completely, since they would have run and passed when the previous version of pack was released.

Reply:
As an alternative the check can be made within the loop.

Reply:
Added a check that `combos` has at least one element with `Pack` equal to "current".